README.adoc

```diff
@@ -12,22 +40,25 @@ ifndef::env-github[]
 image::images/Ui.png[width="600"]
 endif::[]
 
-* This is a desktop Address Book application. It has a GUI but most of the user interactions happen using a CLI (Command Line Interface).
-* It is a Java sample application intended for students learning Software Engineering while using Java as the main programming language.
-* It is *written in OOP fashion*. It provides a *reasonably well-written* code example that is *significantly bigger* (around 6 KLoC)than what students usually write in beginner-level SE modules.
+== Build Status
+
```

Comment:
can remove this line,

Reply:
I think its they auto add this one. Not sure why. Because my intellij dont have

Reply:
HAHAHAHA 🤔

Reply:
wait do you mean remove build status entirely?